frontend/public/components/secret.jsx

```diff
@@ -25,7 +26,8 @@ export const addSecretToWorkload = (kindObj, secret) => {
 
   return {
     callback: () => configureAddSecretToWorkloadModal({ secretName, namespace, blocking: true }),
-    label: 'Add Secret to Workload',
+    // t('workload~Add secret to workload')
+    labelKey: 'workload~Add secret to workload',
```

Comment:
I believe we're using the kind labels here, so Secret would be capitalized.

```suggestion
    labelKey: 'workload~Add Secret to workload',
```